cms/jinja2/templates/pages/methodology_page.html

```diff
@@ -86,6 +86,12 @@
     <div class="ons-grid ons-grid-flex-gap ons-grid-flex-gap--32 ons-js-toc-container">
         <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m">
             {% with toc_title=_("Contents"), toc_aria_label=_("Sections in this page") %}
+
```

Comment:
note: this looks like the remnants of a merge conflict. This section is not needed as of #213

Reply:
:facepalm: This is removed in aa70fd2662ccae54e66d3fdc14986904b3858a61.